statsmodels/robust/robust_linear_model.py

```diff
@@ -466,6 +486,171 @@ def bse(self):
     def chisq(self):
         return (self.params/self.bse)**2
 
+    @cache_readonly
+    def rho(self):
+        return self.model.M.rho(self.sresid)
+
+    @cache_readonly
+    def rho_sum(self):
+        return self.model.M.rho(self.sresid).sum()
+
+    @cache_readonly
+    def deviance(self):
+        return 2 * self.scale**2 * self.rho.sum()
+
+    @cache_readonly
+    def rsquared(self):
+        endog = self.model.endog
+        mod0 = RLM(endog, np.ones(len(endog)), M=self.model.M)
+
+        # Note: fixed_scale is not exposed
+        # rsquared is cached attribute and replicates SAS
+        fixed_scale = False
+        if not fixed_scale:
+            # this replicates an example in SAS documentation
+            res0 = mod0.fit()
+            #rho0 = res0.rho_sum
```

Comment:
Is this comment useful?  If not it should be removed.  If so it needs to be fleshed out.

Also `#` --> `# `

Reply:
I would like to have this in RLM.